### PR TITLE
migrate bridge stuff to zn 2 #Cu-867747xuf

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
based on https://github.com/ComposableFi/composable/pull/2922 for https://app.clickup.com/t/867747xuf

- removed not used bridge stuff
- centauri net works as it worked (removed mounting file toml as directory)
- added oneliner to follow logs of devnet a (not generalised, so it is clear how to proceed) 
- `find` to be in base container 
- enabled logs overrides and offchain indexer in devnet
- nix run .#fmt (seems people do not run fmt)

### CODE REVIEW

File: flake/zombienet.nix [hunk 1]
```diff
@@ -97,11 +117,26 @@
...
         zombienet-picasso-complete = {
           type = "app";
           program = self'.packages.zombienet-picasso-complete;
         };
 
+        zombienet-log-follow = {
+          program = pkgs.writeShellApplication rec {
+            name = "zombienet-log-follow";
+            text = ''
+              docker exec -it composable-devnet-a-1   bash  -c 'LOG=$(find /tmp/ -name "zombie-*" | head --lines=1)/alice.log && tail --follow $LOG'
```
Comment (blasrodri) [inline flake/zombienet.nix:134]: nice

Comment (blasrodri) [reply flake/zombienet.nix:134]: why only Alice?

**Review by blasrodri — APPROVED** [review]

File: flake/zombienet.nix (hunk 1 above)
Comment (dzmitry-lahoda) [reply flake/zombienet.nix:134]: only alice.log node has well know port 9988, other nodes has random ports and will not be published outside

Comment (dzmitry-lahoda) [reply flake/zombienet.nix:134]: so likely nothing will try to connect to bob.log

**Review by andyjsbell — APPROVED** [review]
Thanks @dzmitry-lahoda

**Comment by dzmitry-lahoda** [comment]
moving to fork

**Pull request closed without merging**